lib/studies/reply/mail_card_preview.dart

```diff
@@ -245,6 +233,31 @@ class _MailPreview extends StatelessWidget {
   }
 }
 
+class _PicturePreview extends StatelessWidget {
+  const _PicturePreview();
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 96,
+      child: ListView(
+        scrollDirection: Axis.horizontal,
+        children: [
+          for (var index = 0; index < 4; index++)
+            Padding(
+              padding: const EdgeInsetsDirectional.only(end: 4),
+              child: Image.asset(
+                'reply/attachments/paris_${index + 1}.jpg',
+                package: 'flutter_gallery_assets',
+                isAntiAlias: true,
```

Comment:
What is this needed for?

Reply:
It's not sorry, I forgot I left it in there.